.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=localhost
```

Comment:
This file shouldn't be committed to the repository.

As we go on, the data contained in it will get more and more sensitive and we don't want to have that in the repository.